### PR TITLE
Prepare for DisplayManager

### PR DESCRIPTION
**Pull request by iartemenko**
Add C++ wrapers needed for DisplayManager

### CODE REVIEW

**Comment by andr2000** [comment]
what about bzr instead of tar.bz?

**Comment by iartemenko** [comment]
Done with bzr://

**Comment by iartemenko** [comment]
bump

File: recipes-domx/meta-xt-images-vgpu/recipes-core/properties-cpp/properties-cpp_0.0.1.bb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+SUMMARY = "A VERY SIMPLE CONVENIENCE LIBRARY FOR HANDLING PROPERTIES AND SIGNALS IN C++11."
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-vgpu/recipes-core/properties-cpp/properties-cpp_0.0.1.bb]: WHY CAPS?

File: recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/dbus-cpp_5.0.0.bb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+DEPENDS = "dbus \
+           boost \
+           libxml2 \
+           properties-cpp \
+           process-cpp \
+"
+SRC_URI = "bzr://bazaar.launchpad.net/~phablet-team/dbus-cpp/trunk \
+           file://set_distro.patch \
+"
+SRCREV = "105"
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/dbus-cpp_5.0.0.bb:16]: any reason for this specific rev?

File: recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+   # we define the version to be 2.0.0
++  set(DISTRO_CODENAME "poky")
+   if (${DISTRO_CODENAME} STREQUAL "vivid")
+     set(DBUS_CPP_VERSION_MAJOR 4)
+     set(DBUS_CPP_VERSION_MINOR 3)
+@@ -102,6 +103,5 @@
+ add_subdirectory(examples)
+ add_subdirectory(include)
+ add_subdirectory(src)
+-add_subdirectory(tests)
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch]: do we really need to build tests?

File: recipes-domx/meta-xt-images-vgpu/recipes-core/properties-cpp/properties-cpp_0.0.1.bb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+HOMEPAGE = "HTTPS://LAUNCHPAD.NET/PROPERTIES-CPP"
+
+LICENSE = "LGPLV3"
+LIC_FILES_CHKSUM = "file://COPYING;md5=e6a600fd5e1d9cbde2d983680233ad02"
+
+SRC_URI = "bzr://bazaar.launchpad.net/~phablet-team/properties-cpp/trunk"
+SRCREV = "17"
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-vgpu/recipes-core/properties-cpp/properties-cpp_0.0.1.bb:8]: why this rev?

File: recipes-domx/meta-xt-images-vgpu/recipes-core/process-cpp/process-cpp_2.0.0.bb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+LICENSE = "LGPLv3"
+LIC_FILES_CHKSUM = "file://COPYING;md5=e6a600fd5e1d9cbde2d983680233ad02"
+DEPENDS = "boost \
+           gtest \
+           gmock \
+           properties-cpp \
+"
+
+SRC_URI = "bzr://bazaar.launchpad.net/~phablet-team/process-cpp/trunk"
+SRCREV = "54"
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-vgpu/recipes-core/process-cpp/process-cpp_2.0.0.bb]: why this rev?

File: recipes-domx/meta-xt-images-vgpu/recipes-core/properties-cpp/properties-cpp_0.0.1.bb (hunk 1 above)
Comment (iartemenko) [reply recipes-domx/meta-xt-images-vgpu/recipes-core/properties-cpp/properties-cpp_0.0.1.bb]: Done

File: recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch (hunk 3 above)
Comment (iartemenko) [reply recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch]: No, we do not so removing subdirectory tests from the cmake file

File: recipes-domx/meta-xt-images-vgpu/recipes-core/process-cpp/process-cpp_2.0.0.bb (hunk 5 above)
Comment (iartemenko) [reply recipes-domx/meta-xt-images-vgpu/recipes-core/process-cpp/process-cpp_2.0.0.bb]: All revisions mentioned above are latest/stable. With "norev" or ${AUTOREV} bzt fetches the oldest one.

Comment (andr2000) [reply recipes-domx/meta-xt-images-vgpu/recipes-core/process-cpp/process-cpp_2.0.0.bb]: got it

File: recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch (hunk 3 above)
Comment (andr2000) [reply recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch]: my bad

File: recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+@@ -34,6 +34,7 @@
+   #   - other distros
+   #   - errors
+   # we define the version to be 2.0.0
++  set(DISTRO_CODENAME "poky")
+   if (${DISTRO_CODENAME} STREQUAL "vivid")
+     set(DBUS_CPP_VERSION_MAJOR 4)
+     set(DBUS_CPP_VERSION_MINOR 3)
+@@ -102,6 +103,5 @@
+ add_subdirectory(examples)
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch]: you probably don't need examples

File: recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+ add_subdirectory(include)
+ add_subdirectory(src)
+-add_subdirectory(tests)
+ 
+ enable_coverage_report(benchmark)
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch]: why do you need this?

File: recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch (hunk 6 above)
Comment (iartemenko) [reply recipes-domx/meta-xt-images-vgpu/recipes-core/dbus-cpp/files/set_distro.patch]: you are right, removing tests, docs and examples from dbus-cpp and process-cpp as well

**Review by andr2000 — APPROVED** [review]

**Pull request merged**